local-setup/jupyter/crs_loader.py

```
@@ -0,0 +1,1363 @@
+        # Ensure cleanup pod exists
+        subprocess.run(["kubectl", "delete", "pod", "db-cleanup", "-n", "egov", "--ignore-not-found"],
+                      capture_output=True)
+        subprocess.run(["kubectl", "run", "db-cleanup", "--image=postgres:15", "-n", "egov",
+                       "--restart=Never", "--command", "--", "sleep", "3600"], capture_output=True)
+        subprocess.run(["kubectl", "wait", "--for=condition=Ready", "pod/db-cleanup", "-n", "egov",
+                       "--timeout=60s"], capture_output=True)
+
+        conn_str = f"postgresql://{db_user}:{db_pass}@{db_host}:5432/{db_name}"
+
+        # Delete relationships first, then boundaries
+        result = subprocess.run(
```

Ruff (0.15.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)